packages/api/src/api/videos.ts

```diff
@@ -0,0 +1,34 @@
+import {Router} from 'express';
+import { Video } from '../entities/Video';
+import logger from '../logger';
+
+export const videos = Router();
+
+// Get all videos
+videos.get('', async (req, res) => {
+    try {
+        const videosRepository = req.entityManager.getRepository(Video);
+        const videosList = await videosRepository.findAll();
+
+        res.statusCode = 200;
+        res.send(videosList);
+    } catch (error) {
+        logger.error('There was an issue geting all videos: ', error);
+        res.statusCode = 500;
+        res.send('There was an issue geting all videos');
+    };
+});
+
+// Get video by id
+videos.get('/:video_id', async (req, res) => {
+    const { video_id } = req.params;
+
+    try {      
+        const videosRepository = req.entityManager.getRepository(Video);
+        const video = await videosRepository.findOne({ video_id: Number(video_id) });
```

Comment:
What happens if `Number(video_id)` is `NaN` 🤔🤔🤔

Reply:
This is is resolved by removing the video_id property and using the default id property inherited from the Node entity.  The id property is a string so it will error at the database level if the id isn't in integer format.  At this point we return an invalid id format error.